symfony/phpunit-bridge/3.3/.env.test

```diff
@@ -1,4 +1,5 @@
 # define your env variables for the test env here
 KERNEL_CLASS='App\Kernel'
 APP_SECRET='s$cretf0rt3st'
+APP_DEBUG='true'
```

Comment:
Will maybe be not longer needed when https://github.com/symfony/recipes/pull/530 is merged